recipes/graphviz/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "2.38.0" %}
+
+package:
+  name: graphviz
+  version: {{ version }}
+
+source:
+  fn:  graphviz-{{ version }}.tar.gz
+  url: http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz-{{ version }}.tar.gz
+  md5: 5b6a829b2ac94efcd5fa3c223ed6d3ae
+  patches:
+    - gvconfig_libdir.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  detect_binary_files_with_prefix: True
+
+requirements:
+  build:
+    - cairo  # [linux]
+    - expat  # [linux]
```

Comment:
OK, I take it back. That PR allows the recipe to build without errors, but it doesn't actually fix the links to `expat` in the final binary package -- the system `libexpat.1.dylib` is still used.  For now, let's just leave it alone.

I *think* this means that Mac users will need the XCode command-line tools in order to use graphviz, but that's a reasonable requirement as far as I'm concerned.